example/package.json

```diff
@@ -9,20 +9,22 @@
     "web": "expo start --web"
```

Comment:
Mixture of versions here, can we remove `~` and `^` in order to stick with stricter packages. Might need to remove lockfile and node_modules to re-install fresh.